recipe/meta.yaml

```diff
@@ -29,16 +29,18 @@ requirements:
     - python
 
 test:
+  files:
+    - fib.pyx
   requires:
     - pip
+    # for distutils
+    - setuptools  # [py>=312]
```

Comment:
Why is this needed here but not in the runtime section if the tests aren't being run?

Reply:
Yeah, it was needed for running tests and run_test.py. I'll add tests but they are quite slow

Reply:
Tests are too slow. I added a comment about them. But `setuptools` is still needed for `run_test.py`